src/modules/challenges/Challenges.ts

```diff
@@ -16,6 +16,7 @@ export default class Challenges implements Saveable {
         disableGems: new Challenge('No Gem', 'Disables the usage of Gems for increasing damage multipliers'),
         disableProteins: new Challenge('No Protein', 'Disables the usage of Proteins'),
         slowEVs: new Challenge('Slow EVs', 'Gain EVs 10x slower'),
+        realEvolutions: new Challenge('Real evolutions', 'Your Pokémons go away, when they evolve'),
```

Comment:
Pokémon is also the plural of Pokémon.
```suggestion
        realEvolutions: new Challenge('Real evolutions', 'Your Pokémon go away, when they evolve'),
```